packages/gatsby/cache-dir/develop-static-entry.js

```diff
@@ -2,7 +2,15 @@ import React from "react"
 import { renderToStaticMarkup } from "react-dom/server"
 import { merge } from "lodash"
 import apiRunner from "./api-runner-ssr"
-import testRequireError from "./test-require-error"
+// import testRequireError from "./test-require-error"
+// For some extremely mysterious reason, webpack adds the above module *after*
```

Comment:
@jquense any idea why this might be happening?

Reply:
V weird... Maybe it's the naming throwing off some parser thing hmm